modules/ROOT/pages/create-community.adoc

```diff
@@ -244,6 +246,8 @@ These permissions reflect the current API Community Manager package version. If
 
 == Configure Sharing Settings
 
+Use the following steps to configure custom user settings. To configure users settings automatically with default settings, use the *FIX* buttons in the xref:diagnostics.adoc#user-access-tab[API Community Manager diagnostics user access tab].
```

Comment:
Is this the "guest" user text mentioned earlier? Clarify.
The previous intro mentions members vs. guests, so I am expecting those two topics.
Note previous comment about using the automatic method rather than manual configuration.

Reply:
Updated to clarify.